babel.config.js

```diff
@@ -1,5 +1,13 @@
 module.exports = {
 	presets: [
+		[
+			'@babel/preset-env',
+			{
+				targets: {
+					node: '8.9.0', // Keep this in sync with package.json engines.node
```

Comment:
Should we bump this to 10.x (in both places)?
8.x has been EOL I think since December?

Reply:
That might mean updating the public docs as well: https://wpvip.com/documentation/vip-go/vip-cli/#installing-vip-cli-locally